app/runnable.py

```diff
@@ -58,15 +63,18 @@ def synchronize_keywords(self, input_data: KeywordGradingRequest) -> None:
                 self.problem_dict.pop(problem_id)
                 self.create_problem(input_data)
 
-    def get_tokenized_answer(self, user_answer: str):
-        split_answer = user_answer.strip().split()
-        tokenized_answer = []
+    def get_tokenized_answer(self, user_answer: str) -> List[str]:
+        user_answer = re.sub(r"[^\uAC00-\uD7A3a-zA-Z\s]", "", user_answer)
```

Comment:
검색해보니까 특문 제거용 regex 인 것 같은데, 의미를 알 수있도록 변경해보는건 어떨까요!

```suggestion
        special_char_pattern = r"[^\uAC00-\uD7A3a-zA-Z\s]"
        user_answer = re.sub(special_char_pattern, "", user_answer)
```